issue_release_status.go

```diff
@@ -162,5 +163,7 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 		return errors.Wrapf(err, "commenting on pull request, with DeliveryID '%v'", deliveryID)
 	}
 
+	logger.Debug().Msg("Comment created")
```

Comment:
It might be nice with a bit more details here just to avoid loads of `Comment created` lines.

Maybe the repository and PR number could be added? So it becomes `Comment created on lunar-way-user-service PR 24`?

Reply:
Good idea. https://github.com/lunarway/release-manager-bot/pull/18/commits/6d35b89a96fb8bfcc1a2c6ac4d0b236ce05b6fd4 should resolve this 🙂